index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Responsive Login Page</title>
+    <link rel="stylesheet" href="css/styles.css">
+  </head>
+  
+  <body>
+    
+    <form id="login-form">
+      <h1>Login</h1>
+      Email<br>
+      <input type="email" name="email" placeholder="Enter your email"><br>
+      Password<br>
+      <input type="password" name="password" pattern=".{6,}" required placeholder="6 characters or more please"><br>
+      <input type="submit" id= "submit" value="Login">
+    </form>
+    
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.0/jquery.min.js"></script>
+    <script src="http://ajax.aspnetcdn.com/ajax/jquery.validate/1.14.0/jquery.validate.min.js"></script>
```

Comment:
I think some of the weirdness is coming from this library. You shouldn't need this.